aloscene/frame.py

```diff
@@ -507,10 +510,27 @@ def mean_std_norm(self, mean, std, name) -> Frame:
 
         return tensor
 
+    def norm_meanstd(self, mean_std=None, name=None) -> Frame:
+        """Returns z-norm of the current frame.
+        This method will simply call `frame.mean_std_norm()` with the mean/std property of the frame and the selected name.
+        Instead of a custom mean/std, you can use the resnet norm based on the normalized use of resnet on pytorch.
+        Examples
+        --------
+        >>> frame_resnet = frame.norm_resnet(name="resnet")
+        >>> frame_custom_norm = frame.norm_resnet(name="custom")
+        """
+        if name == "resnet":
+            return self.norm_resnet()
+        elif mean_std is not None and len(mean_std) == 2:
+            return self.mean_std_norm(mean=mean_std[0], std=mean_std[1], name=name)
+        elif self.mean_std is not None:
+            return self.mean_std_norm(mean=self.mean_std[0], std=self.mean_std[1], name=name)
```

Comment:
The attribute `mean_std` is used to store the current normalization parameters. You can delete this